oracle/pkg/updater/updater_test.go

```diff
@@ -285,7 +287,7 @@ func TestUpdater(t *testing.T) {
 			if !bytes.Equal(settlement.builder, c.Commiter.Bytes()) {
 				t.Fatal("wrong builder")
 			}
-			if settlement.amount != 0 {
+			if settlement.amount.Uint64() != 10 {
```

Comment:
why are you getting the Uint64 value here?

Reply:
Thank you for the review! This is for the test. The initial value is 10, so it can be converted to uint64 here.